pkg/apis/nfd/v1alpha1/annotations_labels.go

```diff
@@ -32,6 +32,12 @@ const (
 	// AnnotationNs namespace for all NFD-related annotations.
 	AnnotationNs = "nfd.node.kubernetes.io"
 
+	// ExtendedResourceNs is the namespace for extended resources.
+	ExtendedResourceNs = "nfd.node.kubernetes.io"
```

Comment:
I'm a bit undecided should this be `nfd.node.kubernetes.io` or `feature.node.kubernetes.io` 🧐 Any thoughts? @zvonkok @fidencio @ArangoGutierrez

Reply:
Hmm.  Fair question, and I think a better fit would be `feature.node.kubernetes.io`.

Reply:
`feature.node.kubernetes.io` it is!